src/com/google/javascript/jscomp/AbstractCommandLineRunner.java

```diff
@@ -971,6 +1040,30 @@ protected int doRun() throws FlagUsageException, IOException {
 
     List<String> jsFiles = config.js;
     List<String> moduleSpecs = config.module;
+    List<JsonFileSpec> jsonFiles = null;
+
+    if (config.useJsonStreams) {
+      jsonFiles = parseJsonFilesFromInputStream();
+
+      ImmutableMap.Builder<String, SourceMapInput> inputSourceMaps
+          = new ImmutableMap.Builder<>();
+
+      int jsonInputSourceMapCount = 0;
```

Comment:
You only testing for non-zero of this. You can replace it with a boolean foundJsonInputSourceMap